.github/workflows/release.yml

```diff
@@ -17,37 +17,37 @@ jobs:
     strategy:
       matrix:
         build:
-          - os: ubuntu-20.04
+          - os: ${{ fromJson(github.repository_owner == 'subspace' && github.repository == 'subspace/subspace-cli' && '["self-hosted", "ubuntu-22.04-x86-64", "Linux"]' || '["ubuntu-22.04"]') }}
```

Comment:
Is there any way to abstract this logic out so that it doesn't have to be repeated?

Reply:
Unfortunately, GitHub doesn't support yaml anchors still :(
see https://github.com/orgs/community/discussions/25545
and https://github.com/actions/runner/issues/1182